app/src/components/eventCard/EventCard.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import style from "./style.module.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faSchool,
+  faUsers,
+  faChild,
+  faAtlas,
+  faChalkboardTeacher,
+  faSnowplow,
+  faTableTennis,
+  faHeartbeat,
+  faTheaterMasks,
+  faUserTimes,
+  faUserPlus,
+  faMeteor,
+  faUmbrellaBeach,
+  faDraftingCompass,
+  faChevronRight
+} from "@fortawesome/free-solid-svg-icons";
+
+export default props=>
+{
+  const icons = {
+    'Аттестация'                    : faDraftingCompass ,
```

Comment:
You have the same file changes at another PR, that is bad practice. At future, please, wait for PR with similar changes will merge, or open PR to branch with similar changes :)